projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -233,9 +220,7 @@ public function add_appearance_menu( $wp_admin_themes = false, $wp_admin_customi
 		// Customize on Atomic sites is always done on WP Admin.
 		parent::add_appearance_menu( $wp_admin_themes, true );
 
-		// Add back the Theme Editor submenu.
-		$parent_slug = $wp_admin_themes ? 'themes.php' : 'https://wordpress.com/themes/' . $this->domain;
-		add_submenu_page( $parent_slug, __( 'Theme Editor', 'jetpack' ), __( 'Theme Editor', 'jetpack' ), 'edit_themes', 'theme-editor.php' );
+		add_submenu_page( 'themes.php', esc_attr__( 'Add New Theme', 'jetpack' ), __( 'Add New Theme', 'jetpack' ), 'install_themes', 'theme-install.php', null, 1 );
```

Comment:
Can we fix the fact that the menu is not highlighted when "Add New Theme" is selected? Not introduced by this PR though ;-) 

![](https://cln.sh/7BHd2P+)

Reply:
Any idea about how can we achieve that? Took a quick look and not sure what's causing the submenu to not be highlighted since we're using `add_submenu_page` as expected.

Reply:
No clue...

Reply:
In that case, since this doesn't seem like a trivial fix, let's leave it for a follow-up.